api/server.ts

```diff
@@ -254,6 +257,22 @@ io.on('connection', (socket) => {
             tenderRequestId: offer.tenderRequestId,
           },
         })
+
+        if(tenderRequest && otherOffersForTender)
+          otherOffersForTender.forEach(o =>
+            sendPushNotification({
+              to: [o.supplier.token],
+              title: 'Anbud förkastat',
+              body: `Ditt bud på ${tenderRequest.title} har förkastats. ${offer.supplier.name}s bud har godkänts av följande skäl: ${offer.acceptanceMotivation}`,
+              data: {
+                date: new Date(),
+                type: 'offer',
+                to: [o.supplier.id],
+                id: offer.id,
+                tenderRequestId: tenderRequest?.id,
+              },
+            }) 
+        )
```

Comment:
Formatting looks a bit off here compared to line 247 for example. Spacing between `if` and `(`. Although I'm not sure they have any certain formatting rules.

Reply:
Good spot! My vs code format config differs a lot with how the files are formatted so I tried to avoid formatting entire files and work as neat as possible, I fixed this now!

Reply:
Awesome. Feel free to merge and open a PR toward parent :)